src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js

```diff
@@ -61,7 +61,7 @@ const ActivityThreadReplies = ({
                     getUserProfileUrl={getUserProfileUrl}
                     mentionSelectorContacts={mentionSelectorContacts}
                     onDelete={onDelete}
-                    onEdit={onEdit}
+                    onCommentEdit={onEdit}
```

Comment:
Why is this change needed?

Reply:
Because `BaseComment` no longer has an onEdit prop. It takes `onCommentEdit` and `onAnnotationEdit` and resolves the differences for the handlers, etc. which make use of `onEdit`.